src/client/minimap.cpp

```diff
@@ -181,19 +184,26 @@ Minimap::Minimap(Client *client)
 	this->m_ndef    = client->getNodeDefManager();
 
 	m_angle = 0.f;
+	m_current_mode_index = 0;
 
 	// Initialize static settings
 	m_enable_shaders = g_settings->getBool("enable_shaders");
 	m_surface_mode_scan_height =
 		g_settings->getBool("minimap_double_scan_height") ? 256 : 128;
 
+	// Initialize minimap modes
+	addMode(MINIMAP_TYPE_OFF,     "hidden", 0, "");
+	addMode(MINIMAP_TYPE_SURFACE, "surface mode, Zoom x1", 256, "");
+	addMode(MINIMAP_TYPE_SURFACE, "surface mode, Zoom x2", 128, "");
+	addMode(MINIMAP_TYPE_SURFACE, "surface mode, Zoom x4", 64, "");
+	addMode(MINIMAP_TYPE_RADAR,   "radar mode, Zoom x1", 512, "");
+	addMode(MINIMAP_TYPE_RADAR,   "radar mode, Zoom x2", 256, "");
+	addMode(MINIMAP_TYPE_RADAR,   "radar mode, Zoom x4", 128, "");
```

Comment:
Can this be translatable?

Reply:
Yes ! Sure ! Change made and old labels are reused so translations are yet working again :)